packages/yoroi-extension/chrome/extension/ergo-connector/api.js

```diff
@@ -259,16 +263,43 @@ export async function connectorSignTx(
     keyLevel: wallet.getParent().getPublicDeriverLevel(),
```

Comment:
What about P2S inputs that include keys we might need to be able to sign with? Would we have to look into the context extension or into the ergoscript to be able to figure out what those keys are to be able to sign those inputs?